sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -1264,6 +1265,29 @@ class DDLSuite extends QueryTest with SharedSQLContext with BeforeAndAfterEach {
     }
   }
 
+  test("create table using cluster by without schema specification") {
```

Comment:
s/cluster/clustered? Even an uppercase variant CLUSTERED BY as in the other descriptions.